CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.16.0-beta.0 / 2019-01-14
+
+* [ENHANCEMENT] Update prometheus/common and add support for --log.format (#1658)
+* [ENHANCEMENT] Add CORS support to API v2 (#1667)
+* [ENHANCEMENT] Support HTML, URL title and custom sounds for Pushover (#1634)
+* [ENHANCEMENT] [amtool] Support for assuming first label is alertname in silence add and query (#1693)
```

Comment:
I would say this is a bugfix, since I believe it was already listed as possible in the documentation